### DISCUSSION TITLE
Nextflow performance slow startup on k8s

### DISCUSSION THREAD

**Discussion by namxle** [nextflow-io/nextflow#4569] [Q&A]
### Discussed in https://github.com/nextflow-io/nextflow/discussions/4547

<div type='discussions-op-text'>

<sup>Originally posted by **namxle** November 29, 2023</sup>
Hello, currently, I am running Nextflow on a Kubernetes cluster and trying to improve its performance. While going through the logs, I noticed that within these steps, it took nearly 10 minutes, whereas running it in a local environment took only a few seconds.

<img width="1030" alt="image" src="https://github.com/nextflow-io/nextflow/assets/103374580/4b5dad4c-7ca2-420a-bf4f-8170e039b45a">

The nextflow version is 23.04.3.

Can anyone help me optimize this or provide any reasons behind this? Thanks a lot.</div>

**Comment by bentsherman** [comment]
Duplicate of #4547
